src/support/timestamp.c

```diff
@@ -298,7 +298,7 @@ __wt_time_aggregate_validate(
      *
      */
 
-    if (ta->oldest_start_ts > ta->newest_stop_ts)
+    if (ta->newest_stop_ts != WT_TS_NONE && ta->oldest_start_ts > ta->newest_stop_ts)
```

Comment:
Can we add a comment?

Reply:
Added a comment [here](https://github.com/wiredtiger/wiredtiger/pull/12471/commits/9ebb0097450bb30bfbff845fe2be0a39d04a78c0).